DelegateFun/Sorter.Tests/SortUtilityTests.cs

```diff
@@ -1,4 +1,5 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
 

```

Comment:
```suggestion

using Sorter.SortUtil;
```

Save some typing if you just import Sorter.SortUtil at the top instead.